x-pack/examples/reporting_example/public/components/app.tsx

```diff
@@ -70,28 +84,72 @@ export const ReportingExampleApp = ({
     return {
       layout: {
         id: constants.LAYOUT_TYPES.PRESERVE_LAYOUT,
-        selectors: getDefaultLayoutSelectors(),
       },
       relativeUrls: ['/app/reportingExample#/intended-visualization'],
       objectType: 'develeloperExample',
       title: 'Reporting Developer Example',
     };
   };
 
+  const getPDFJobParamsDefaultV2 = (): JobParamsPDFV2 => {
+    return {
+      version: '8.0.0',
+      layout: {
+        id: constants.LAYOUT_TYPES.PRESERVE_LAYOUT,
+      },
+      locatorParams: [
+        { id: REPORTING_EXAMPLE_LOCATOR_ID, version: '0.5.0', params: { myTestState: {} } },
+      ],
+      objectType: 'develeloperExample',
+      title: 'Reporting Developer Example',
+      browserTimezone: moment.tz.guess(),
+    };
+  };
+
+  const getPNGJobParamsDefaultV2 = (): JobParamsPNGV2 => {
+    return {
+      version: '8.0.0',
+      layout: {
+        id: constants.LAYOUT_TYPES.PRESERVE_LAYOUT,
+      },
+      locatorParams: {
+        id: REPORTING_EXAMPLE_LOCATOR_ID,
+        version: '0.5.0',
+        params: { myTestState: {} },
```

Comment:
Is there a way to update the test state?

Reply:
Oh, I see it is only updated internally, and can be seen in the generated report. 

As an example, should this also include some client-side state, such as simply the value of a click counter? Maybe for a later PR?

Reply:
That sounds like a good improvement! I'll open an issue for it after this gets merged!